okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/LeakDetector.java

```diff
@@ -108,4 +101,15 @@ private void logLeak(Optional<RuntimeException> stackTrace) {
             this.stackTrace = stackTrace;
         }
     }
+
+    static boolean isTraceLoggingEnabled() {
+        return log.isTraceEnabled();
+    }
+
+    interface Reporter {
+
+        void onLeakDetected(Optional<RuntimeException> stacktrace);
+
+        boolean createStackTrace();
```

Comment:
this name was a little weird to me - kinda sounded like it might be actually doing the creation under the hood, but actually it's just a pure getter used in if condition. `shouldCreateStackTrace` or `isStackTraceCreationEnabled`?